internal/pkg/describe/env.go

```diff
@@ -0,0 +1,82 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package describe
+
+import (
+	"encoding/json"
+	//"encoding/json"
+	"fmt"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/archer"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/aws/session"
+
+	//"github.com/aws/amazon-ecs-cli-v2/internal/pkg/aws/session"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/store"
+)
+
+type Environment struct {
+	Name       string `json:"name"`
+	Production bool   `json:"production"` // add to instances used elsewhere (ie project_show)?
+	Region     string `json:"region"`
+	AccountID  string `json:"accountID"`
+}
+
+type EnvApp struct {
+	Name        string `json:"name"`
+	Description string `json:"description"`
+}
+
+type EnvTag struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+
+type EnvSummary struct {
+	Environment  []*Environment `json:"environment"`
```

Comment:
If this is just a single environment, this shouldn't be a slice, correct? Also, if this is also just extending the `Environment` type in line 17, you can just [embed](https://golang.org/doc/effective_go.html#embedding) it.

Reply:
Oh also -- since this struct actually provides more information than the existing `Environment`, what do you think about renaming that one to "EnvironmentSummary" and calling the new one `Environment`? This would be consistent with how ListResource API calls return a ResourceSummary, wheres `GetResource` API calls return the full Resource itself in the aws SDK. (eg. see [ListPipelines](https://github.com/aws/aws-sdk-go/blob/master/service/codepipeline/api.go#L8780-L8789) versus [GetPipeline](https://github.com/aws/aws-sdk-go/blob/master/service/codepipeline/api.go#L7077-L7085))

Reply:
Ah, I see now. Thanks!